docs/example-app.rst

```diff
@@ -17,3 +17,32 @@ We encourage you to give it a try with a few commandline calls:
 A screenshot of the example app:
 
 .. image:: _static/screenshot.jpg
+
+Postgres
+########
+
+You can use Postgres as the source database.  To do this, you will need to have `Docker <https://docker.com/>`_ installed.
```

Comment:
You will need to have => You can use
Docker not being the only option to install postgres :)